build/miengine.settings.targets

```diff
@@ -8,28 +8,48 @@
   <Import Project="version.settings.targets" />
 
   <Choose>
-    <When Condition="'$(Configuration)' == 'Lab.Debug' Or '$(Configuration)' == 'Lab.Release'">
+    <When Condition="'$(Lab)' == 'true'">
         <PropertyGroup>
             <!--NOTE:DevDivPublicKey.snk comes from src\\tools\\devdiv\\FinalPublicKey.snk -->
             <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)..\Keys\DevDivPublicKey.snk</AssemblyOriginatorKeyFile>
             <DelaySign>true</DelaySign>
+            <Lab>true</Lab>
         </PropertyGroup>
     </When>
     <Otherwise>
         <PropertyGroup>
             <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)..\Keys\ExternalKey.snk</AssemblyOriginatorKeyFile>
             <DelaySign>false</DelaySign>
+            <Lab>false</Lab>
         </PropertyGroup>
     </Otherwise>
   </Choose>
   
+  <Choose>
+    <When Condition="$(Configuration.Contains('.Desktop'))">
+      <PropertyGroup>
+        <IsCoreClr>false</IsCoreClr>
+        <NugetPackagesDirectory>$(NugetPackagesDirectory).Desktop</NugetPackagesDirectory>
+        <!-- Skip resolution from project.json -->
+        <ResolveNuGetPackages>false</ResolveNuGetPackages>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <PropertyGroup>
+        <DefineConstants>$(DefineConstants);CORECLR</DefineConstants>
```

Comment:
This is slightly confusing because the non-desktop configuration runs against the desktop CLR also. Maybe we should use DESKTOP_CFG instead?